app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -73,7 +73,11 @@ def verify_credentials(args)
       )
       secret_access_key = MiqPassword.try_decrypt(secret_access_key)
 
-      !!raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, :assume_role => assume_role)
+      begin
+        !!raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, validate = true, :assume_role => assume_role)
+      rescue
```

Comment:
`validate = true`? Is it just `true` or is it supposed to be `:validate => true`?

Reply:
Nope, as of [here](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/manager_mixin.rb#L80), but maybe the allocation is unnecessary and it can be just `true`.

Reply:
Just for readability, seeing just true as an arg doesn't really mean anything without going to the method description and counting args

Reply:
This is why I like kwargs for stuff like this

Reply:
We might don't want the exception handling here, testing ...

Reply:
Yeah, better without the handling, the task fails and the exception message is actually helpful.

Reply:
:+1: done